storybook/src/components/Header/Header.stories.tsx

```diff
@@ -29,64 +73,62 @@ export const ForSubBrand: Story = {
 
 export const WithAppName: Story = {
   args: {
-    appName: 'Aan de Amsterdamse grachten',
+    appName: 'Onderzoek en Statistiek',
   },
 }
 
-export const WithLinks: Story = {
+export const WithMenu: Story = {
   args: {
-    links: (
-      <PageMenu alignEnd wrap={false}>
-        <PageMenu.Link href="#">Contact</PageMenu.Link>
-        <PageMenu.Link href="#">Mijn Amsterdam</PageMenu.Link>
-        <PageMenu.Link href="#" icon={SearchIcon}>
-          Zoeken
-        </PageMenu.Link>
+    menu: (
+      <PageMenu alignEnd>
+        <PageMenu.Item>
+          <PageMenu.Link href="#">Menu link 1</PageMenu.Link>
+        </PageMenu.Item>
+        <PageMenu.Item>
+          <PageMenu.Link href="#">Menu link 2</PageMenu.Link>
+        </PageMenu.Item>
+        <PageMenu.Item>
+          <PageMenu.Link href="#">Menu link 3</PageMenu.Link>
+        </PageMenu.Item>
       </PageMenu>
```

Comment:
Perhaps you could use real links like the other stories? 
```suggestion
        <PageMenu.Link href="#">Contact</PageMenu.Link>
        <PageMenu.Link href="#">Mijn Amsterdam</PageMenu.Link>
        <PageMenu.Link href="#">Nieuws</PageMenu.Link>
```
And it might help to test 5 links, which is the advised maximum amount of links in this navigation element.